test/runner.js

```diff
@@ -1,20 +1,23 @@
-
 /**
  * Module dependencies.
  */
-
+ 
```

Comment:
Try installing one of the [editorconfig plugins](http://editorconfig.org/#download) to automatically enable whitespace trimming for this project.